### PR TITLE
Replaced all the css into tailwind css on OutboundLink component

### PR DESCRIPTION
**Pull request by subhajit20**
# Description of changes

Replaced all the css into tailwind css on OutboundLink component
Issue - #1725 

Before:
Before

After:
After

# Issue Resolved

Fixes #NA

### CODE REVIEW

**Comment by subhajit20** [comment]
hey @kylemh , is there any changes needed?

**Comment by subhajit20** [comment]
> Unit tests are failing!

done

File: components/OutboundLink/OutboundLink.js [hunk 1]
```diff
@@ -48,7 +47,7 @@ function OutboundLink({
 
       <ScreenReaderOnly>Opens in new window</ScreenReaderOnly>
 
-      {hasIcon && <ExternalLinkIcon className={styles.externalLinkIcon} />}
+      {hasIcon && <ExternalLinkIcon className="fill-current relative my-0 mx-1 bottom-0 w-3" />}
```
Comment (kylemh) [inline components/OutboundLink/OutboundLink.js:50]:
```suggestion
      {hasIcon && <ExternalLinkIcon className="!fill-current relative my-0 mx-1 bottom-0 w-3" />}
```

Comment (subhajit20) [reply components/OutboundLink/OutboundLink.js]: done

**Pull request merged**